### DISCUSSION TITLE
Dictionaries

### DISCUSSION THREAD

**Discussion by gc1984** [simjanos-dev/LinguaCafe#280] [Ideas]
Will you consider using the libretranslate api?

Although the use of the Deepl api is free, you still have to signup with creditcard and i don't feel comfortable with that.

And staying with the selfhost spirit, libretranslate would be a nice option i think.

https://github.com/LibreTranslate/LibreTranslate

What do you think?

**Comment by simjanos-dev** [comment]
> Although the use of the Deepl api is free, you still have to signup with creditcard and i don't feel comfortable with that.

Not sure if this was you, but someone also had the same opinion on discord

I did not know about this tool before. I think this is probably worse than DeepL, but it is self-hosted, and there's no reason not to add it as an option. 

I'll add [argos-translate](https://github.com/argosopentech/argos-translate) directly, without using the LibreTranslate wrapper. 

I will have to test it, and it will probably also have to be installed as a package like a language on the admin page if it is more than few dozen of MB.

I'll add it when I'll have the time, this would be a really valuable addition to linguacafe. :) 

Thank you so much!

( I'm also wondering how good Llama3 would work as a translator. )

**Comment by simjanos-dev** [comment]
Hi!

LibreTranslate support was added in v0.14. MyMemory is also supported, which is not self hosted, but free.
